packages/prover/src/cli/cmds/start/options.ts

```diff
@@ -0,0 +1,81 @@
+import {NetworkName, networksChainConfig} from "@lodestar/config/networks";
+import {LightNode} from "../../../interfaces.js";
+import {CliCommandOptions} from "../../../utils/command.js";
+
+export type StartArgs = {
+  port: number;
+  network: string;
+  "execution-rpc": string;
```

Comment:
use use `executionUrls` and make it an array which we may support later

Reply:
Renamed to `execution-url`. I don't see a reason to have it array for now. If we make it plural for now user can mislead to behavior.